src/pages/Home/Home.js

```diff
@@ -1,14 +1,17 @@
-import React from 'react'
+import React from 'react';
 import Heading from '../../components/Heading'
 import WorkoutForm from '../../components/WorkoutForm/WorkoutForm'
+import NavBars from '../../components/NavBar/NavBar';
+
+function Home () {
```

Comment:
Can we keep declaring functions with `const` for consistency please?

Reply:
Fixed

Reply:
Have you pushed it?

Reply:
So I'm keeping track for all of them

Reply:
👍